chalice/app.py

```diff
@@ -8,33 +8,34 @@
 sys.path.insert(0, pkg_root)  # noqa
 
 import monitor
+from monitor import events
 
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
-stage = os.getenv('DSS_INFRA_TAG_STAGE')
-app = chalice.Chalice(app_name=f"dss-monitor-{stage}")
+app = chalice.Chalice(app_name=f"dss-monitor-dev")
```

Comment:
I think chalice will attach the stage suffix on deployment, so `dss-monitor` is sufficient here.